app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -631,7 +631,7 @@ class PatientSummaryUpdateTest {
   }
 
   @Test
-  fun `when contact doctor button is clicked, then load patient teleconsultation information`() {
+  fun `when contact doctor button is clicked and only one phone number is present, then load patient teleconsultation information`() {
```

Comment:
Instead of describing the implementation detail only one phone number is present, can we reword this to reflect the real world usage? Ex: `only one doctor is available for teleconsultation`.